docs/faker/file.md

```diff
@@ -0,0 +1,45 @@
+## File fake generator
```

Comment:
Do we have file fake generator allowed in the cli. If not I think is a good idea to add it and document